ros/api/v0/hosts.py

```diff
@@ -151,11 +152,12 @@ def build_sort_expression(self, order_how, order_method):
 class HostDetailsApi(Resource):
     profile_fields = {
         'host_id': fields.String(attribute='inventory_id'),
-        'performance_record': fields.String,
-        'display_performance_score': fields.String,
         'rating': fields.Integer,
-        'recommendation_count': fields.Integer,
```

Comment:
Instead of `recommendation_count`, will it be good to rename this to `number_of_recommendations`?